### PR TITLE
Use accessible-icon primitive

### PR DESCRIPTION
**Pull request by chaance**
@benoitgrelard This one's for you, homie

### CODE REVIEW

File: components/DocsPage.tsx [hunk 1]
```diff
@@ -65,8 +65,7 @@ export function DocsPage({ children }: { children: React.ReactNode }) {
                   ':focus': { boxShadow: 'none' },
                 }}
               >
-                <VisuallyHidden>Radix Documentation (Alpha)</VisuallyHidden>
-                <RadixLogo aria-hidden />
+                <RadixLogo label="Radix Homepage" />
```
Comment (benoitgrelard) [inline components/DocsPage.tsx:68]: The label seemed different here, are we ok with changing it?

File: pages/blog/index.tsx [hunk 2]
```diff
@@ -33,22 +33,7 @@ export default function Home() {
...
-                overflow: 'hidden',
-                clip: 'rect(0, 0, 0, 0)',
-                whiteSpace: 'nowrap',
-                border: 0,
-              }}
-            >
-              Stitches homepage
-            </span>
-            <RadixLogo />
+            <RadixLogo label="Radix Homepage" />
```
Comment (benoitgrelard) [inline pages/blog/index.tsx:36]: Label has changed, probably wasn't right, but may not be radix homepage?

File: pages/design-system/index.tsx [hunk 3]
```diff
@@ -19,22 +19,7 @@ export default function DesignSystemHome() {
...
-                overflow: 'hidden',
-                clip: 'rect(0, 0, 0, 0)',
-                whiteSpace: 'nowrap',
-                border: 0,
-              }}
-            >
-              Stitches homepage
-            </span>
-            <RadixLogo />
+            <RadixLogo label="Radix Homepage" />
```
Comment (benoitgrelard) [inline pages/design-system/index.tsx:22]: Same question for label

File: components/DocsPage.tsx (hunk 1 above)
Comment (chaance) [reply components/DocsPage.tsx:68]: Yeah, it was wrong before. It should be the label for the link, which goes to the homepage.

File: pages/blog/index.tsx (hunk 2 above)
Comment (chaance) [reply pages/blog/index.tsx:36]: I think it is. Right now everything is redirected to the Radix intro page which is defacto home. We can change these later as needed.

**Pull request merged**